jsx.js

```diff
@@ -13,7 +13,78 @@ module.exports = {
   },
 
   rules: {
-    'jsx-quotes': [2, 'prefer-double']
+    'jsx-quotes': [2, 'prefer-double'],
+    'react/boolean-prop-naming': [0],
+    'react/button-has-type': [0],
+    'react/default-props-match-prop-types': [2],
+    'react/destructuring-assignment': [0],
+    'react/forbid-component-props': [0],
+    'react/forbid-dom-props': [0],
+    'react/forbid-elements': [0],
+    'react/forbid-prop-types': [2],
+    'react/function-component-definition': [2, {namedComponents: 'arrow-function'}],
+    'react/jsx-boolean-value': [0],
+    'react/jsx-child-element-spacing': [2],
+    'react/jsx-closing-bracket-location': [2, 'line-aligned'],
+    'react/jsx-closing-tag-location': [2],
+    'react/jsx-curly-brace-presence': [2, {props: 'never', children: 'never'}],
+    'react/jsx-curly-newline': [2],
+    'react/jsx-curly-spacing': [2, 'never'],
+    'react/jsx-equals-spacing': [0],
+    'react/jsx-filename-extension': [0],
+    'react/jsx-first-prop-new-line': [2],
+    'react/jsx-fragments': [2, 'syntax'],
+    'react/jsx-handler-names': [0],
+    'react/jsx-indent': [2, 2, {checkAttributes: true, indentLogicalExpressions: true}],
+    'react/jsx-indent-props': [2, 2],
+    'react/jsx-max-depth': [0],
+    'react/jsx-max-props-per-line': [2],
```

Comment:
I'd prefer to leave this disabled, I commonly put multiple props on a single line for e.g. SVG elements
(like `<circle cx={50} cy={50} r={25}/>`), and files can get pretty long this way, often with just a handful of characters on many of its lines.

Reply:
i agree about such edge cases, but i feel there might be more less readable instances if we disable it

Reply:
Wait, you want all JSX tags to be spread over rmultiple lines?

So you'd prefer seeing:

```jsx
<div onClick={onClick} style={styles.div}>
  <a href="google.com" target="_blank">
    <Icon name="google" type="filled" />
  </a>
</div>
```

like this:

```jsx
<div
  onClick={onClick}
  style={styles.div}
>
  <a
    href="google.com"
    target="_blank"
  >
    <Icon
      name="google"
      type="filled"
    />
  </a>
</div>
```

I'd be strongly opposed.

Reply:
I would indeed prefer the first, so i rest my case

Reply:
Is there value on specifying the maximum number of props, or just disable it?

Reply:
I think we should disable it. Line length is good enough to dictate the max props, I think.